app/src/main/res/menu/action_bar.xml

```diff
@@ -88,6 +88,8 @@
           android:orderInCategory="100"
           app:showAsAction="never">
         <menu>
+            <item android:id="@+id/action_debug_mode"
```

Comment:
I'm going to hide this dev menu behind a settings flag as it looks a bit messy.

Reply:
(Just an FYI - you don't need to do anything. I've been fiddling with the menus in my Bric4 branch to add custom entries for instrument ops).